src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -145,15 +148,15 @@ export const applicantGenderSchema = {
             'ui:title': `What's ${nameWording(
               formData,
               undefined,
-              undefined,
+              false,
```

Comment:
Setting to `false` so that 'you/r' isn't automatically capitalized. This change was carried out in several places.